class/class.2.1/hello.cpp

```diff
@@ -0,0 +1,84 @@
+#include <iostream>
+#include <fstream>
+#include <unordered_map>
+#include <string>
+
+const std::string FILENAME = "user.txt";
+
+class UserStats {
+public:
+    void load() {
+        std::ifstream infile(FILENAME);
```

Comment:
it is more reasonable to provide file name as input argument. It will allow to test your class on test files instead of prod file. So you will be able to provide your FIILENAME constant for the main program while for testing yyouu can use user_test.txt or any other file